src/keyfile.c

```diff
@@ -235,6 +235,8 @@ static void init_pref_groups(void)
 		"use_gio_unsafe_file_saving", TRUE);
 	stash_group_add_boolean(group, &file_prefs.keep_edit_history_on_reload,
 		"keep_edit_history_on_reload_125", FALSE);
```

Comment:
This needs to be renamed back to `keep_edit_history_on_reload` without the `_125` suffix to reset the default value.